var/spack/repos/builtin/packages/robodoc/package.py

```diff
@@ -0,0 +1,31 @@
+# Copyright 2013-2021 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+
+from spack import *
+
+
+class Robodoc(AutotoolsPackage):
+    """ROBODoc is program documentation tool."""
+
+    homepage = "http://www.xs4all.nl/~rfsber/Robo/index.html"
```

Comment:
```suggestion
    homepage = "https://www.xs4all.nl/~rfsber/Robo/index.html"
```
Or perhaps use the URL this redirects to (i.e., https://rfsber.home.xs4all.nl/Robo/index.html)?